scripts/build_js.js

```diff
@@ -107,6 +107,28 @@ function compileCPP(packageName) {
     }
 }
 
+// eslint-disable-next-line no-unused-vars
+function compileRust() {
+    const base_dir = path.join(__dirname, "..", "rust", "perspective", "arrow-data-slice");
```

Comment:
should "arrow-data-slice" not be "arrow_accessor"?

Reply:
good catch - that was outdated build code which I've replaced with better integration into our existing build system.